test/e2e_node/remote/utils.go

```diff
@@ -60,14 +59,25 @@ providers:
     - "*.pkg.dev"
     defaultCacheDuration: 1m`
 
+func getCNIURL() string {
+	cniArch := "amd64"
+	if builder.IsBuildForArm64() {
```

Comment:
TODO: And when we're using existing builds?

Seems like what we want is the target node architecture.

Reply:
Correct, this checks whether the targetArch is "linux/arm64"

Reply:
AFAICT this checks the architecture we're _Building_ for, but you should be able to run node_e2e without building?

Reply:
(or else, is at least confusingly named if not only checking for builds)

Reply:
This is a parameter passed in by user to indicate the target node architecture, I will update the name to `IsTargetArchArm64`